Export/Skills/sup_dex.txt

```diff
@@ -299,6 +296,9 @@ local skills, mod, flag, skill = ...
 	},
 #mods
 
+#skill SupportAdditionalCooldown
```

Comment:
That doesn't seem to be the right place if we're going through them alphabetically.

Reply:
Alright, we're going through them based on the in-game order.